test/functional/spam2_controller_test.rb

```diff
@@ -228,4 +249,70 @@ def teardown
         assert_equal "Only admins and moderators can batch delete.", flash[:error]
         assert_redirected_to "/dashboard"
     end
+    #flag Nodes
+    test 'Anyone user should be able to flag a note' do
```

Comment:
Great job writing the tests...lets fix the grammar here 'Any user  should...' or 'Anyone should...'

Reply:
Sorrry :sweat_smile:  for that, Done. Thanks!!